cloudinit/sources/__init__.py

```diff
@@ -302,6 +302,11 @@ def persist_instance_data(self):
         instance_data.update(
             self._get_standardized_metadata())
         instance_data['ds']['_doc'] = EXPERIMENTAL_TEXT
+        # Add merged cloud.cfg and sys info for jinja templates and cli query
+        instance_data['cfg'] = copy.deepcopy(self.sys_cfg)
+        instance_data['cfg']['_doc'] = 'Merged cloud-init system config'
+        instance_data['sys_info'] = util.system_info()
```

Comment:
Do we need to encode any of this or is this output json acceptable?

Reply:
So far does not look like it. the values from system_info are all being represented as strings